mart/transforms/torchvision_ref.py

```diff
@@ -58,8 +58,10 @@ def __call__(self, image, target):
         classes = [obj["category_id"] for obj in anno]
         classes = torch.tensor(classes, dtype=torch.int64)
 
-        segmentations = [obj["segmentation"] for obj in anno]
-        masks = convert_coco_poly_to_mask(segmentations, h, w)
+        masks = None
+        if anno and "segmentation" in anno[0]:
```

Comment:
If there are not annotations, this would fail. I think you should add a guard clause before to verify that there is at least 1 annotation to proceed with this logic.

Reply:
Thanks. Fixed in 89cb1de2b03cb6d5edab746e8454aa4d2fc37491